### PR TITLE
Add missing domains from various service lists

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: assets/services.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
       "name": "9GAG",
       "rules": [
         "||9cache.com^",
+        "||9g.ag^",
```
Comment (Sergey-Lyapin) [inline assets/services.json:17]: Hello @L8X
Where is this domain being used? I can't find it.

File: assets/services.json [hunk 2]
```diff
@@ -2016,6 +2031,7 @@
         "||ext-twitch.tv^",
         "||jtvnw.net^",
         "||ttvnw.net^",
+        "||twitch.amazon.com^",
```
Comment (Sergey-Lyapin) [inline assets/services.json:2034]: This domain redirects to gaming.amazon.com. I think it's better not to add it here.

File: assets/services.json [hunk 3]
```diff
@@ -1357,12 +1361,14 @@
         "||instagify.com^",
         "||instagmania.com^",
         "||instagor.com^",
+        "||instagr.am^",
```
Comment (Alex-302) [inline assets/services.json:1364]: instagr.am, instagram.net
just old domains, which redirect to the main. No need to block it.

File: assets/services.json [hunk 4]
```diff
@@ -2465,10 +2482,12 @@
       "id": "rockstar_games",
       "name": "Rockstar games",
       "rules": [
+        "||rockstar.com^",
```
Comment (Alex-302) [inline assets/services.json:2485]: Where used?

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by Freebase394** [comment]
Could you check #282

File: assets/services.json [hunk 5]
```diff
@@ -2465,10 +2482,12 @@
...
+        "||rockstar.games^",
```
[the remark this answers is not in the document]
Comment (Alex-302) [reply assets/services.json:2486]: main - rockstargames.com The purpose of the filter is block sites/services. Any url shortener can redirect to rockstargames.com, but this is not a reason to block them.

File: assets/services.json [hunk 6]
```diff
@@ -2430,6 +2446,7 @@
         "||dradis-prod.rdatasrv.net^",
         "||pvp.net^",
         "||rgpub.io^",
+        "||riot.com^",
```
Comment (Alex-302) [inline assets/services.json:2449]: redirection. Remove.

File: assets/services.json (hunk 2 above)
Comment (Alex-302) [inline assets/services.json:2034]: amazon shoud not be blocked

File: assets/services.json [hunk 7]
```diff
@@ -1820,13 +1827,21 @@
...
+        "||rbx.cn^",
+        "||rbxcdn.com^", 
+        "||rbxcdn.net^", 
+        "||rbxinfra.com^",
+        "||rbxinfra.net^",
         "||roblox.cn^",
         "||roblox.com^",
+        "||roblox.de^",
         "||roblox.qq.com^",
+        "||roblox.uk^",
```
[the remark this answers is not in the document]
Comment (Alex-302) [reply assets/services.json:1841]: This filter is not censorship. We block specific **services**, not any domain related to company

File: assets/services.json [hunk 8]
```diff
@@ -1820,13 +1827,21 @@
...
+        "||blox.com^",
+        "||rbx.com^",
+        "||rbx.cn^",
+        "||rbxcdn.com^", 
+        "||rbxcdn.net^", 
+        "||rbxinfra.com^",
+        "||rbxinfra.net^",
         "||roblox.cn^",
         "||roblox.com^",
+        "||roblox.de^",
```
Comment (Alex-302) [inline assets/services.json:1839]: redirection. Remove.

File: assets/services.json (hunk 4 above)
Comment (Alex-302) [reply assets/services.json:2485]: just redirection using short name. Remove.

File: assets/services.json (hunk 2 above)
Comment (Alex-302) [reply assets/services.json:2034]: amazon should not be blocked here.

File: assets/services.json [hunk 9]
```diff
@@ -1820,13 +1827,21 @@
       "id": "roblox",
       "name": "Roblox",
       "rules": [
+        "||blox.com^",
+        "||rbx.com^",
+        "||rbx.cn^",
```
Comment (Alex-302) [inline assets/services.json:1832]: redirection. Remove.

File: assets/services.json [hunk 10]
```diff
@@ -1357,12 +1361,14 @@
...
         "||instagmania.com^",
         "||instagor.com^",
+        "||instagr.am^",
         "||instagram-brand.com^",
         "||instagram-engineering.com^",
         "||instagram-help.com^",
         "||instagram-press.com^",
         "||instagram-press.net^",
         "||instagram.com^",
+        "||instagram.net^",
```
Comment (Alex-302) [inline assets/services.json:1371]: redirection. Remove.

File: assets/services.json (hunk 3 above)
Comment (Alex-302) [inline assets/services.json:1364]: redirection. Remove.

File: assets/services.json [hunk 11]
```diff
@@ -920,8 +922,10 @@
         "||facebook.cc^",
         "||facebook.com^",
         "||facebook.design^",
+        "||facebook.gg^",
         "||facebook.hu^",
         "||facebook.in^",
+        "||facebook.me^",
```
Comment (Alex-302) [inline assets/services.json:928]: redirection. Remove.

File: assets/services.json [hunk 12]
```diff
@@ -920,8 +922,10 @@
         "||facebook.cc^",
         "||facebook.com^",
         "||facebook.design^",
+        "||facebook.gg^",
```
Comment (Alex-302) [inline assets/services.json:925]: redirection. Remove.

File: assets/services.json (hunk 1 above)
Comment (Alex-302) [reply assets/services.json:17]: In any case this is redirection to 9gag.com, not owned by 9gag. Please remove.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

File: assets/services.json [hunk 13]
```diff
@@ -1820,13 +1820,19 @@
...
+        "||rbxcdn.com^", 
+        "||rbxcdn.net^", 
+        "||rbxinfra.com^",
+        "||rbxinfra.net^",
         "||roblox.cn^",
         "||roblox.com^",
         "||roblox.qq.com^",
         "||robloxcdn.com^",
         "||robloxdev.cn^",
         "||rbxadder.com^",
```
Comment (Sergey-Lyapin) [inline assets/services.json]: @L8X 
Please delete the unnecessary comma

**Pull request merged**